### ISSUE TITLE
plugin-postcss - Error: "Cannot find package 'cssnano'" with npm >= 7

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#833]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
When trying out plugin-postcss to update the TailwindCSS example in the Greenwood Getting Started project, was getting this error on build
```sh
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'cssnano' imported from /Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/plugin-postcss/src/index.js
    at packageResolve (internal/modules/esm/resolve.js:655:9)
    at moduleResolve (internal/modules/esm/resolve.js:696:18)
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:810:11)
    at Loader.resolve (internal/modules/esm/loader.js:88:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:241:28)
    at Loader.import (internal/modules/esm/loader.js:176:28)
    at importModuleDynamically (internal/modules/esm/translators.js:114:35)
    at exports.importModuleDynamicallyCallback (internal/process/esm_loader.js:30:14)
    at PostCssResource.optimize (file:///Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/plugin-postcss/src/index.js:74:8)
    at async file:///Users/owenbuckley/Workspace/project-evergreen/repos/greenwood-getting-started/node_modules/@greenwood/cli/src/config/rollup.config.js:46:29 {
  code: 'PLUGIN_ERROR',
  pluginCode: 'ERR_MODULE_NOT_FOUND',
  plugin: 'greenwood-html-plugin',
  hook: 'writeBundle'
}
```

## Details
This is likely due to the way that Yarn vs NPM (v7) handle hoisting and deduping dependencies.  Although [`@greenwood/cli` depends on `cssnano`](https://github.com/ProjectEvergreen/greenwood/blob/v0.20.1/packages/cli/package.json#L33), as they are sibling packages, NPM v7 is just walking up the tree recursively child to parent, not also child <> sibling -> parent, thus resolving it from **plugin-postcss** it will not be found.

You can see how it shows up in _node_modules_
<img width="595" alt="Screen Shot 2021-12-24 at 11 37 28 AM" src="https://user-images.githubusercontent.com/895923/147365064-85d65000-914c-49c2-b4eb-66d5fbc9b382.png">
<img width="707" alt="Screen Shot 2021-12-24 at 11 37 33 AM" src="https://user-images.githubusercontent.com/895923/147365065-2e9fec2b-6505-4ae2-9c6e-6bbdc3192249.png">

TL;DR if we are going to actually call upon `cssnano` in **plugin-postcss,** we should properly attribute it as a `dependency`
